### PR TITLE
update copy and link

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds a utm link and shortens the banner text per feedback

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -71,14 +71,14 @@ var siteSettings = {
     },
     announcementBar: {
       id: "biweekly-demos",
-      content: "Join us on May 14th for the dbt Cloud Launch Showcase event. Discover the latest innovations and watch live demos! ",
+      content: "Join us on May 14th and 15th for the dbt Labs Launch Showcase",
```
Comment (mirnawong1) [inline website/docusaurus.config.js]:
```suggestion
      content: "Join us on May 14th and 15th for the dbt Labs Launch Showcase event.",
```

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -71,14 +71,14 @@ var siteSettings = {
...
       backgroundColor: "#047377",
       textColor: "#fff",
       isCloseable: true,
     },
     announcementBarActive: true,
     announcementBarLink:
-      "https://www.getdbt.com/resources/webinars/dbt-cloud-launch-showcase",
+      "https://www.getdbt.com/resources/webinars/dbt-cloud-launch-showcase/?utm_medium=internal&utm_source=docs&utm_campaign=q2-2025_dbt-cloud-launch-showcase_aw&utm_content=____&utm_term=all___",
```
Comment (nghi-ly) [inline website/docusaurus.config.js:81]: non-blocking sugg: consider using a URL shortener

**Review by nghi-ly — APPROVED** [review]
:shipit:

File: website/docusaurus.config.js (hunk 2 above)
Comment (mirnawong1) [reply website/docusaurus.config.js:81]: oh yea it's a utm link that jakki provided in the thread!

**Pull request merged**